core/src/test/java/org/infinispan/eviction/impl/EvictionWithConcurrentOperationsTest.java

```diff
@@ -374,6 +385,87 @@ public void testScenario7() throws Exception {
       assertInMemory(key1, "v2");
    }
 
+   // Data is written to container but before releasing orderer the entry is evicted. In this case the entry
+   // should be passivated to ensure data is still around
+   public void testEvictionDuringWrite() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
Maybe we also need to test the case where eviction removes the entry but a write/remove acquires the orderer and writes/removes the entry before the eviction can acquire the orderer? Or maybe it only makes a difference with passivation?

Reply:
It only matters with passivation. testWriteDuringEviction of the other class should be handling that.